test/models/AppModel/test.js

```diff
@@ -0,0 +1,142 @@
+const {
+  expect, AppModel, faker, dropDatabase,
+} = require('test/testHelper');
+const { AppFactory } = require('test/factories/');
+const _ = require('lodash');
+
+describe('App Model', async () => {
+  describe('On getOne', async () => {
+    let app, result;
+    beforeEach(async () => {
+      app = await AppFactory.Create();
+    });
+    it('Should check names for identity', async () => {
+      result = await AppModel.getOne({ name: app.name });
+      expect(result.app).to.deep.eq(app._doc);
+    });
+    it('Should check that the error exists', async () => {
+      result = await AppModel.getOne({ name: faker.random.string() });
+      expect(result.error).is.exist;
```

Comment:
expect(result.error).to.be.exist     (if you will completely write off the tests from the second project, I will write comments on each, write yourself)